vocabularies/telephone-type.ttl

```diff
@@ -0,0 +1,41 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix sdo: <http://www.schema.org/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix tsl: <http://linked.data.gov.au/def/telephone-type/> .
+@prefix vcard: <http://www.w3.org/2006/vcard/ns#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+
+<http://linked.data.gov.au/def/telephone-type> a owl:Ontology,
+        skos:ConceptScheme ;
+    dcterms:created "2020-03-25"^^xsd:date ;
+    dcterms:creator <http://linked.data.gov.au/org/gsq> ;
+    dcterms:modified "2020-03-25"^^xsd:date ;
+    dcterms:publisher <http://linked.data.gov.au/org/gsq> ;
+    dcterms:source "Derived from Australian Standard AS 4590.1:2017" ;
+    skos:definition "The code and description that differentiates the type of telecommunications device based on the transmission service connected to the telephone network."@en ;
+    skos:prefLabel "Telephone Type"@en ;
+    skos:hasTopConcept 
+        vcard:Cell ,
+        vcard:Voice .
+
+<http://linked.data.gov.au/org/gsq> a sdo:Organization ;
+    sdo:name "Geological Survey of Queensland" .   
+
+vcard:Cell a skos:Concept , vcard:TelephoneType ;
+    skos:altLabel "Cell"@en,
+        "Cellular"@en ;
+    skos:definition "A portable telephone that can make and receive calls over a radio frequency link while the user is moving within a telephone service area."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/telephone-type> ;
+    skos:notation "01" ;
```

Comment:
These notations were from the AS4590 standard.
If our source now is vcard are the notations still valid?

Reply:
Sure. Notation is there to allow you to associate a term with something else, such as a code used in a particular environment. You might have to add a note as to what that environment is, perhaps in the ConceptScheme scopeNote you could say this.